cmake/CMakeLists.txt

```diff
@@ -377,6 +384,40 @@ IF(COLLADA_SUPPORT AND BUILD_CONVERT)
 	ENDIF()
 ENDIF()
 
+if(HDF5_SUPPORT)
+    IF("${HDF5_INCLUDE_DIR}" STREQUAL "")
+		MESSAGE(STATUS "No HDF5 include directory specified")
+	ElSE()
+		SET(HDF5_INCLUDE_DIR "${HDF5_INCLUDE_DIR}" CACHE FILEPATH "HDF5 header files")
+	ENDIF()
+	
+    IF("${HDF5_LIBRARY_DIR}" STREQUAL "")
+		MESSAGE(STATUS "No HDF5 library directory specified")
+	ElSE()
+		SET(HDF5_LIBRARY_DIR "${HDF5_LIBRARY_DIR}" CACHE FILEPATH "HDF5 library files")
+	ENDIF()
+	
+	
+	MESSAGE(STATUS HDF5_INCLUDE_DIR "${HDF5_INCLUDE_DIR}")
+    IF(WIN32)
+        # An additional lib- prefix is added by HDF5 to denote static libraries
+        SET(HDF5_LIBRARIES 
+            "${HDF5_LIBRARY_DIR}/libhdf5_cpp.lib"
+            "${HDF5_LIBRARY_DIR}/libhdf5.lib"
+            "${HDF5_LIBRARY_DIR}/libzlib.lib"
+			"${HDF5_LIBRARY_DIR}/libsz.lib"
+			"${HDF5_LIBRARY_DIR}/libaec.lib")
+    ELSE()
+        SET(HDF5_LIBRARIES 
+            "${HDF5_LIBRARY_DIR}/hdf5_cpp.lib"
```

Comment:
The extension for library files on non-windows is not .lib (it's .a for static, .so for dynamic, .dylib for dynamic on OSX). For now maybe just change to .a